cpp/daal/src/algorithms/dbscan/dbscan_distributed_input_types.cpp

```diff
@@ -249,19 +251,17 @@ services::Status DistributedInput<step3Local>::check(const daal::algorithms::Par
     const size_t nDataBlocks = dcPartialData->size();
     DAAL_CHECK_EX(nDataBlocks > 0, ErrorIncorrectNumberOfInputNumericTables, ArgumentName, partialDataStr());
```

Comment:
I would suggest to handle case, when nBlocks == 0 in the same manner as the empty partialData tables.